lib/Doctrine/ODM/PHPCR/UnitOfWork.php

```diff
@@ -100,6 +101,18 @@ class UnitOfWork
     private $documentIds = array();
 
     /**
+     * @var array
+     */
+    private $documentUuids = array();
```

Comment:
Ideally this would be better named `$oidToUuidMap`, but to be consistent this naming is fine for now I guess. But it should be commented.